src/Equinor.ProCoSys.IPO.Query/GetSavedFiltersInProject/GetSavedFiltersInProjectQueryHandler.cs

```diff
@@ -42,5 +52,7 @@ public class GetSavedFiltersInProjectQueryHandler: IRequestHandler<GetSavedFilte
 
             return new SuccessResult<List<SavedFilterDto>>(savedFilterDtos);
         }
+
+        private static SuccessResult<List<SavedFilterDto>> EmptyList() => new(new List<SavedFilterDto>());

```

Comment:
would prefer to be explicit here: new SuccessResult<List<SavedFilterDto>>(new List<SavedFilterDto>()); 
(but ok to let it be)